apps/ui/components/search/SingleSearchReservationUnitCard.tsx

```diff
@@ -196,21 +197,27 @@ const StyledTag = styled(Tag)<{ $status: "available" | "no-times" | "closed" }>`
   }
 `;
 
-const StatusTag = (ru: {
+const StatusTag = ({
+  data,
+  id,
+}: {
   data: { closed: boolean; availableAt: string };
+  id?: Maybe<number>;
 }): JSX.Element => {
   const { t } = useTranslation();
-  const { closed, availableAt } = ru.data;
+  const { closed, availableAt } = data;
 
   if (closed) {
     return (
-      <StyledTag $status="closed">{t("reservationUnitCard:closed")}</StyledTag>
+      <StyledTag $status="closed" id={`status-tag-${id}`}>
```

Comment:
Since we get the actual `id` string as a prop after the change mentioned in the reply to the first comment,  I'll just use it directly here as the html `id`.
But I see your point, since as it is now we define another value using the `id`-prop, to use as the html `id` here.